API_Wiki_Call.py

```diff
@@ -2,16 +2,37 @@
 """"Using Wikipedia's API to return the events that happened on the day chosen by the user."""
 
 import requests
-from pprint import pprint
+import API_Response
 
-# user_date is currently hardcoded, this will take a variable later that is passed to it from ui.py.
-user_date = 'October 9'
 
-query = {'action':'opensearch', 'namespace':0, 'search': user_date, 'limit':1}
+def API_Wiki_Call(user_date):
 
+    query = {'action':'opensearch', 'namespace':0, 'search': user_date, 'limit':1}
+    url ='https://en.wikipedia.org/w/api.php'
 
-url ='https://en.wikipedia.org/w/api.php'
+    wiki_api_response = API_Response.API_Response()
 
-data = requests.get(url, params = query).json()
+    """ This will try to get the request from wiki, if it fails for some reason, a message will print, 
+     and return the error, api manager should handle errors from the API call.  Assuming it is successful, 
+     and there is an item now in wiki_date_page_selection, that will then be stored in extracted_page_url, and returned. 
+     If there is nothing returned, it will print a message and return None. """
+     
+    try:
 
-pprint(data[3])
\ No newline at end of file
+            wiki_date_page_selection = requests.get(url, params = query).json()
+            wiki_data_check = wiki_date_page_selection[3]
+
+            if wiki_data_check:
+                data = wiki_date_page_selection
+                wiki_api_response.data = data
```

Comment:
Is the entire response in a usable format by the UI?

Reply:
That is not quite clear at this point. Currently, I have a landing page with a date selector and a button. Once I have passing info from the date selector to the Movie API and displaying results from that one API we will then be able to meet as a group and go over how we want to pass and show the other API's results.